server/src/repositories/config.repository.ts

```diff
@@ -69,12 +68,18 @@ const getEnv = (): EnvData => {
     }
   }
 
-  const globalEnabled = parseBoolean(process.env.IMMICH_METRICS, false);
-  const hostMetrics = parseBoolean(process.env.IMMICH_HOST_METRICS, globalEnabled);
-  const apiMetrics = parseBoolean(process.env.IMMICH_API_METRICS, globalEnabled);
-  const repoMetrics = parseBoolean(process.env.IMMICH_IO_METRICS, globalEnabled);
-  const jobMetrics = parseBoolean(process.env.IMMICH_JOB_METRICS, globalEnabled);
-  const telemetryEnabled = globalEnabled || hostMetrics || apiMetrics || repoMetrics || jobMetrics;
+  const telemetryInclude =
+    process.env.IMMICH_TELEMETRY_INCLUDE === 'all'
+      ? Object.values(ImmichTelemetries).join(',')
+      : process.env.IMMICH_METRICS_INCLUDE;
+  const includedTelemetries = asSet(telemetryInclude, [] as ImmichTelemetries[]);
```

Comment:
I think we should let `asSet` convert the string input to a set. If the set has a single item and it is `all`, then replace it. This will also prevent you from needing to do this weird string manipulation right here.

Reply:
Yeah that's a good point